content/news/doomhammer-tournament-announcement.yaml

```diff
@@ -0,0 +1,7 @@
+---
+kind: article
+title: "Announcing Hecthor Doomhammer's Red Alert Tournament"
+author: "Hecthor Doomhammer"
```

Comment:
It's a bit weird to have this "written" by Hecthor, referring to himself as his own good friend.